Add acks (#717)
mlr-org/mlr3book: book/index.qmd

Readers may choose to selectively read sections in this chapter depending on your use case (i.e., if you have domain-specific problems to tackle), or to use these as introductions to new domains to explore.
Finally, @sec-fairness will introduce algorithmic fairness, which includes specialized measures and methods to identify and reduce algorithmic biases.

## Acknowledgments

As well as the editors and contributing authors, many others have contributed to this book.
We would like to acknowledge Stefan Coors for creating many of the images in the book, as well as Daniel Saggau, Jakob Richter, and Marvin Böcker for contributions to earlier drafts the book.

> Finally we would like thank Munich Center for Machine Learning (MCML) for supporting many of the contributing authors on this book as well as NSF.

Author: ```suggestion
Finally we would like thank Munich Center for Machine Learning (MCML) for supporting many of the contributing authors on this book as well as the National Science Foundation.
```

> Finally we would like thank Munich Center for Machine Learning (MCML) for supporting many of the contributing authors on this book as well as the National Science Foundation.

Reviewer: This sounds like the MCML is supporting NSF...

I would phrase this simply as:
"Many of the contributing authors were supported by the Munich Center for Machine Learning (MCML) and some by the National Science Foundation (NSF)."

> Finally we would like thank Munich Center for Machine Learning (MCML) for supporting many of the contributing authors on this book as well as the National Science Foundation.

Author: ```suggestion
Many of the contributing authors were supported by the Munich Center for Machine Learning (MCML) and some by the National Science Foundation (NSF).
```
